frontend_server/test/server_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:args/src/arg_results.dart';
+import 'package:frontend_server/server.dart';
+import 'package:test/test.dart';
+
+class MockedCompiler implements CompilerInterface {
+  var compileCallback;
+  var recompileDeltaCallback;
+  var acceptLastDeltaCallback;
+  var rejectLastDeltaCallback;
+  var invalideCallback;
+
+  MockedCompiler({this.compileCallback, this.recompileDeltaCallback,
+    this.acceptLastDeltaCallback, this.rejectLastDeltaCallback,
+    this.invalideCallback});
+
+  @override
+  compile(String filename, ArgResults options) =>
```

Comment:
Interesting! I looked at dart:mockito, but it has not been mirrored to chromium.googlesource.com yet. I asked Bill to help with getting it mirrored. After that I will take closer look at that again.

Functions looks cool, but how do I use them with ```@override``` annotation?